docs/learn/learn-async-backing.md

```diff
@@ -7,100 +7,111 @@ keywords: [parachains, slots, backing, parablock]
+  during that block and the merkelized hashes of the unchanged values.
 
 :::
 
-For more information on the validity and availability process, be sure to visit the
-[parachain protocol](../learn/learn-parachains-protocol.md) page.
-
-## Synchronous Backing on Polkadot
-
-With synchronous backing, there was only about a single relay chain block (6-second window), to
-complete the parablock candidate generation and backing process. This was tightly coupled to the
-relay chain's progress, where blocks had to be created within this window.
-
-The main limitation of synchronous backing is that parablock validation is tightly coupled to the
-relay chain's progression on a 1-1 basis, meaning every parablock must be generated and backed
-within six seconds. This time limit reduces the amount of data a collator has time to add to each
-block.
-
-A particular parablock, `P1`, would **only** be valid for backing at relay chain parent `R1 + 1` and
-subsequently, be included at `R1 + 2` should it be backed successfully. Essentially, a parablock is
-rushing to be backed and included within this two-block window due to the inherent requirement for
-synchrony between the parachain and relay chain.
-
-## Asynchronous Backing on Polkadot
-
-With asynchronous backing, the window of time is customizable and will most likely sit around the
-6-18 second range. It also introduces a parameter to aid in defining the maximum amount of ancestor
-blocks, which allows for a parablock to be backed later in the future, enabling more computational
-and storage time per block. The context for the latest parablock is derived from the unincluded
-segment of block ancestors upon which the newest parablock is built.
-
-These blocks can be prepared in anticipation of being included later rather than keeping in sync
-with the relay chain's progress 1-1. The block can be in a different stage from another block being
-built, as long as it abides by the parameters set forth by the asynchronous backing configuration.
+Asynchronous backing brings the following changes to the parachain protocol:
+
+- parablock generation is decoupled from the backing process, this avoids rushing through the
+  backing process in six seconds and gives more time to collators to properly use blockspace,
+  decreasing the probability of generating empty parablocks
+- parachains can generate blocks and have them cashed (or stacked) into **unincluded segments** of
+  parablock ancestors (i.e. parablocks that are seconded but not yet backed by paravalidators and
+  included in the relay chain) rather than ancestors included in the relay chain state.
+
+These changes allow to generate parablock `P + 1` soon after parablock `P` has been generated and
+cashed into the unicluded segment within the first six-second time window. At the same time,
+parablock ancestor `P - 1`, pushed to the unincluded segment before `P`, is backed and included in
+the relay chain.
+
```

Comment:
We probably need to talk more about the prospective parachains subsystem here, along with the concept of parachain fragments.  Prospective parachains keep track of the various paths that a parachain's state can be in, of which each prospective parablock may be in a different state.

```suggestion
These changes allow for a parablock `P + 1`, to begin the generation process soon after parablock `P` has been generated and appended into the unincluded segment within the first six-second time window (the span of one relay chain block). At the same time, parablock ancestor `P - 1`, could be pushed to the unincluded segment before `P`, is backed and included in the relay chain.

```